Python/Product/PythonTools/PythonTools/Project/PackageInstallInfoBar.cs

```diff
@@ -91,7 +93,9 @@ public PackageInstallInfoBar(PythonProjectNode projectNode)
                         Action = PackageInstallInfoBarActions.Ignore,
```

Comment:
I want to keep the log events for workspace and projects separately. So, one log event for projects and one logging event for workspaces. This will give us another datapoint about how many people are using project vs workspace, as well the similarities or dissimilarities. 

Doesn't have to be in this PR--file an issue?